### DISCUSSION TITLE
My application has been approved, but I still haven't received the permissions.

### DISCUSSION THREAD

**Discussion by dawangzelin** [community/community#148052] [GitHub Education] [answered]
### Hi everyone,

My application has been approved, but I still haven't received the permissions. Could you please help me with this? I am in a bit of a hurry and I apologize for taking up your time. I would be very grateful if you could assist me!

**Comment by webbyahtisham** [comment]
what kind of your application??

**Reply by dawangzelin** [reply]
I'm sorry, I didn't understand what you meant. What application are you referring to?

**Comment by davevad93** [comment, marked as answer]
Hi @dawangzelin ,

GitHub no longer provide manual reviews, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @dawangzelin was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
